gorjana/flows/brokering/shipsi.md

```diff
@@ -1,38 +1,125 @@
+1. **Fulfillment Location:** Shipsi identifies the designated fulfillment location based on pre-configured settings.
+2. **Email Notification:** An email containing the order details is sent to the relevant store for fulfillment purposes.
+3. **Celigo Integration:** Stores are mapped to unique location identifiers within Celigo, a data integration platform.
+4. **Shopify Order Data:** The Shopify Order JSON file includes the associated Shipsi location ID under the "shipping_lines" field. **(A sample JSON structure will be added)**
+5. **Fulfillment Update:** Once the driver completes delivery, Shipsi marks the order as fulfilled, and this update is reflected within Shopify.
 
 ## Proposed OMS Fulfillment Process
 
 1. **OMS Fulfillment App:** Orders placed with the Shipsi delivery option are automatically displayed within the designated fulfillment application of the OMS.
-2. **Store Fulfillment:**  Store associates utilize the OMS fulfillment app to mark orders as picked, packed, and shipped.
-3. **Fulfillment Posting:**  The OMS records the order as fulfilled and transmits the pertinent data to NetSuite, the enterprise resource planning (ERP) system.
-4. **Shopify Fulfillment:**  Shipsi maintains its role of marking orders as fulfilled within Shopify.
+2. **Store Fulfillment:** Store associates utilize the OMS fulfillment app to mark orders as picked, packed, and shipped.
+3. **Fulfillment Posting:** The OMS records the order as fulfilled and transmits the pertinent data to NetSuite, the enterprise resource planning (ERP) system.
+4. **Shopify Fulfillment:** Shipsi maintains its role of marking orders as fulfilled within Shopify.
 
 ## Exclusions
 
-* **Bespoke Edition Products:**  Products designated as "Bespoke Edition" will remain ineligible for fulfillment through Shipsi.
+- **Bespoke Edition Products:** Products designated as "Bespoke Edition" will remain ineligible for fulfillment through Shipsi.
 
 ## Conclusion
 
 This integrated approach between Shipsi, Shopify, and the upcoming OMS aims to streamline the purchase-to-fulfillment journey for customers selecting Shipsi's same-day delivery option. By leveraging automation and real-time data exchange, this collaboration enhances efficiency and customer satisfaction.
+
+<details>
+
+<summary>Facility Identification mappings for Shipsi Location</summary>
+
+**Enumeration**
+
+```xml
+<Enumeration description="Shipsi Unique code" enumId="SHIPSI_FULFMENT_LOC" enumCode="SHIPSI_FULFILLMENT_LOCATION" enumTypeId="FACILITY_IDENTITY"/>
+```
+
+**Facility Mapping with Shipsi Locations**
+
+```xml
+<FacilityIdentification facilityId="42" facilityIdenTypeId="SHIPSI_FULFMENT_LOC" fromDate="2024-05-01 05:31:34.056" idValue="5fa1e191c545cd91588b456d" />
+<FacilityIdentification facilityId="96" facilityIdenTypeId="SHIPSI_FULFMENT_LOC" fromDate="2024-05-01 05:31:34.056" idValue="63f37d3fc545cd9c318e1da6" />
```

Comment:
Please remove the spaces before closing tags.

Reply:
Remove: Spaces removed before closing tags.